File: app/src/main/java/App/repositories/retrofit/responses/VentasResponse.java
package App.repositories.retrofit.responses;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import App.repositories.room.entities.Ventas;

public class VentasResponse {



    @SerializedName("status")
    @Expose
    private String status;

    @SerializedName("message")
    @Expose
    private String message;

    @SerializedName("data")
    @Expose
    private List<Ventas> ventas = null;

    public VentasResponse(String status, String message, List<Ventas> ventas) {
        this.status = status;
        this.message = message;
        this.ventas = ventas;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<Ventas> getVentas() {
        return ventas;
    }

    public void setVentas(List<Ventas> ventas) {
        this.ventas = ventas;
    }
    
}

File: app/src/main/java/App/repositories/retrofit/responses/VentasResponse.kt
package App.repositories.retrofit.responses

import App.repositories.room.entities.Ventas
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class VentasResponse(@field:Expose @field:SerializedName("status") var status: String, @field:Expose @field:SerializedName("message") var message: String, ventas: List<Ventas>?) {

    @SerializedName("data")
    @Expose
    var ventas: List<Ventas>? = null

    init {
        this.ventas = ventas
    }
}